docs/excel/excel-add-ins-advanced-concepts.md

```diff
@@ -23,6 +23,10 @@ While you'll likely use the Excel JavaScript API to develop the majority of func
 
 - [Document](/javascript/api/office/office.document): The **Document** object provides the `getFileAsync()` method, which you can use to download the Excel file where the add-in is running.
 
+The following image illustrates when you might use the Excel JavaScript API or the Common APIs.
+
+![Image of the differences between the Excel JS API and Common APIs](../images/excelvscommon.png)
```

Comment:
This might be confusing since "cells" aren't an Excel object. Consider using ranges instead.